docs/runbooks/VirtApiRESTErrorsBurst.md

```diff
@@ -1,36 +1,58 @@
+<!-- Edited by Jiří Herrmann, 7 Nov 2022 -->
+
 # VirtApiRESTErrorsBurst
 
 ## Meaning
 
-Virt-api REST calls are failing in a very high rate.
-Alert will be fired when more than 80% of the REST calls failed in virt-api for the last 5 minutes.
+More than 80% of REST calls have failed in the `virt-api` pods in the last 5 minutes.
 
 ## Impact
 
-Very high rate of failed REST calls to virt-api could lead to slow response / execution of API calls,
-and can even lead to API calls being completely dismissed.
+A very high rate of failed REST calls to `virt-api` might lead to slow response and execution of API calls, and potentially to API calls being completely dismissed.
 
-Currently running VM workloads should not be affected.
+However, currently running virtual machine workloads are not likely to be affected. 
 
 ## Diagnosis
 
-- Set the environment variable `NAMESPACE`
-  ```
-  export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
-  ```
-
-- Check to see how many running virt-api pods exist.
-  ```
-  kubectl -n $NAMESPACE get pods -l kubevirt.io=virt-api
-  ```
-
-- For these pods, try to see the logs with `kubectl logs` and pod status via `kubectl describe`
+1. Set the `NAMESPACE` environment variable:
+   ```
+   $ export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
+   ```
+
+2. Obtain the list of `virt-api` pods on your deployment:
+   ```
+   $ kubectl -n $NAMESPACE get pods -l kubevirt.io=virt-api
+   ```
+
+3. Check the `virt-api` logs for error messages:
+   ```
+   $ kubectl logs -n  $NAMESPACE <virt-api-pod-name>
+   ```
+
+4. Obtain the details of the `virt-api` pods:
+   ```
+   $ kubectl describe -n $NAMESPACE <virt-api-pod-name>
+   ```
+
+5. Check if any problems occurred with the nodes. For example, they might be in a `NotReady` state:
+   ```
+   $ kubectl get nodes
+   ```
+
+6. Check the status of the `virt-api` deployment to find out more information. The following commands provide the associated events and show if any problems occurred, such as crashing pods or failures to pull images:
```

Comment:
This step really should be broken into 2 steps:

6. Check the status of the `virt-api` deployment: 
command block

7. Obtain the details of the  ... deployment:
command block